build/sign.props

```diff
@@ -10,4 +10,14 @@
       <Authenticode>$(SigningIdentity)</Authenticode>
     </FilesToSign>
   </ItemGroup>
-</Project>
\ No newline at end of file
+
+  <!-- Include this segment to debug signing -->
```

Comment:
Should we remove this section before check in?

Reply:
I think it should be checked in. For future debugging convenience.

However, including this un-commented would cause "dotnet restore" to fail on azure pipeline. For some reasons I didn't bother to investigate it has issues pulling the package from microbuild source. (maybe credentials)
Also any developers external to microsoft won't be able to build if it is un-commented.

Reply:
That's exactly the reason I suggest remove it and not check this part in - "Also any developers external to microsoft won't be able to build if it is un-commented."

For an open-source project, we shouldn't include random things for internal debugging usage - it will simply confuse the readers. You can simply keep this section in your onenote - how many times do we really need to debug microbuild signing?

Reply:
ok, removing in next revision